### ISSUE TITLE
Expose artist name on all years/shows/etc endpoints

### ISSUE THREAD

**Issue by switz** [RelistenNet/RelistenApi#20]

**Comment by alecgorge** [comment]
@switz can you add a bit more details? would more artist info be useful?

**Comment by switz** [comment]
Yeah it would be useful on a few of the endpoints, such as the future `source` and `track` endpoints. I just need it for relisten sonos where each request is independent. Using a local artist slug/name cache at the moment. Not a huge deal.
